espresso/environment/tx_helpers.go

```
@@ -129,3 +130,39 @@ func RunSimpleL2Burn(ctx context.Context, t *testing.T, system *e2esys.System) {
 
 	cancel()
 }
+
+// RunSimpleMultiTransactions sends numTransactions simple L2 transactions
+// from Bob's account with a bunch of random data applied to each transaction.
+func RunSimpleMultiTransactions(ctx context.Context, t *testing.T, system *e2esys.System, numTransactions int) []*types.Receipt {
+	senderKey := system.Cfg.Secrets.Bob
+	senderAddress := system.Cfg.Secrets.Addresses().Bob
+	l2Seq := system.NodeClient(e2esys.RoleSeq)
+	nonce, err := l2Seq.NonceAt(ctx, senderAddress, nil)
+	if err != nil {
+		require.NoError(t, err, "failed to get nonce for account %s", senderAddress)
+	}
+
+	ch := make(chan *types.Receipt, numTransactions)
+	for i := 0; i < numTransactions; i++ {
+		go (func(ch chan *types.Receipt, i int, nonce uint64) {
+			receipt := helpers.SendL2Tx(t, system.Cfg, l2Seq, senderKey, func(opts *helpers.TxOpts) {
+				opts.Nonce = nonce + uint64(i)
+				opts.Gas = 100_000
+
+				// opts.Data = make([]byte, 256)
```

Can we remove these commented lines?

---

Or maybe they should be uncommented?

---

I initially attempted to add some arbitrary data to try and make the transactions larger.  However, this is a dynamic fee transaction, and this was resulting in an error, so I opted to just utilize simple transactions instead.

So you're correct in that unused commented out code should be removed.